### PR TITLE
DOC: added all projects toctree to ophyd

### PR DESCRIPTION
**Pull request by mrakitin**
## Description

Add all projects toctree to the ophyd documentation.

## Motivation and Context

Motivated by https://github.com/NSLS-II/docs/issues/73.

## How Has This Been Tested?

Tested locally by building the documentation (required to do `sudo apt-get install graphviz`).

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
this looks good to me, but will sphinx fail then if you had to install graphviz?
I did some digging around, the travis file builds the docs, clones an nsls2-ci repo and eventually runs this file:
https://github.com/NSLS-II/nsls2-ci/blob/master/make-docs.sh

does this mean we should add `sudo apt-get install graphviz` to the travis.yml file or something similar?
@danielballan may have suggestions?

**Comment by jrmlhermitte** [comment]
nevermind, i see it in `DOCS_PIP_DEPS` in travis.yml file. looks good!

File: doc/source/index.rst [hunk 1]
```diff
@@ -69,3 +69,27 @@ other control systems.
...
+
+   bluesky <https://nsls-ii.github.io/bluesky>
+   ophyd <https://nsls-ii.github.io/ophyd>
+
+.. toctree::
+   :hidden:
+   :caption: Data Access and Management
+
+   databroker <https://nsls-ii.github.io/databroker>
+   amostra <https://nsls-ii.github.io/amostra>
```
Comment (danielballan) [inline doc/source/index.rst]: As amostra and datamuxer and both unmaintained, I think we should not advertise them so prominently (for now). Would you mind removing them?

**Comment by mrakitin** [comment]
I updated doctree by excluding unsupported codes and adding caproto docs link (see https://github.com/NSLS-II/docs/issues/82 for motivation).

**Pull request merged**